e107_admin/userclass2.php

```diff
@@ -391,7 +391,7 @@ public function initialPage()
 				    <td>".$frm->select('init_class_stage', $initClassStages, $sel_stage)."<span class='field-help'>".UCSLAN_46."</span>
 
 				    </td></tr></table>
-				    <div class='buttons-bar'>".	$frm->admin_button('set_initial_classes','no-value','create',LAN_UPDATE)."</div>";
+				    <div class='buttons-bar' center>".	$frm->admin_button('set_initial_classes','no-value','create',LAN_UPDATE)."</div>";
```

Comment:
It's this 
`<div class='buttons-bar center'>`
not this right?   
`<div class='buttons-bar' 'center'>`

Oops No 3